.travis.yml

```diff
@@ -4,12 +4,11 @@ jdk:
 - openjdk7
```

Comment:
I've made it, if anybody needs it is in https://github.com/travis-ci/travis-ci/issues/4178#issuecomment-119321586.